packages/react-core/src/demos/examples/Nav/NavFlyout.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {
+  Page,
+  Nav,
+  NavList,
+  NavItem,
+  PageSidebar,
+  PageSidebarBody,
+  PageSection,
+  PageSectionVariants,
+  Menu,
+  MenuContent,
+  MenuList,
+  MenuItem
+} from '@patternfly/react-core';
+import DashboardHeader from '../DashboardHeader';
+
+export const VerticalPage: React.FunctionComponent = () => {
+  const [activeItem, setActiveItem] = React.useState<number | string>(0);
+
+  const onNavSelect = (
+    _event: React.FormEvent<HTMLInputElement>,
+    selectedItem: {
+      groupId: number | string;
+      itemId: number | string;
+      to: string;
+    }
+  ) => setActiveItem(selectedItem.itemId);
+
+  const onMenuSelect = (_event: React.MouseEvent | undefined, itemId: string | number | undefined) => {
```

Comment:
Are the `undefined` types necessary for these params?

Reply:
I am getting this error if I don't put `undefined` there: `Type '(_event: React.MouseEvent, itemId: string | number) => void' is not assignable to type '(event?: MouseEvent<Element, MouseEvent> | undefined, itemId?: string | number | undefined) => void`

when I tried writing the parameters as optional: `(_event?: React.MouseEvent, itemId?: string | number)`, then I get a `SyntaxError: Unexpected token (30:31)`, because of the `?` symbol, which is forbidden in Javascript files (maybe this has to do something with the build tool, that is creating the demo docs), even when I import the file as a typescript file: `ts file='./examples/Nav/NavFlyout.tsx' isFullscreen` I get the syntax error